network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -468,6 +468,33 @@ class Homepage(FoundationMetadataPageMixin, Page):
         blank=True
     )
 
+    quote_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='quote_image',
+    )
+
+    quote_text = models.CharField(
+        max_length=500,
+        default='',
```

Comment:
I had to expliclity set default to empty string otherwise makemigrations would prompt

<img width="1038" alt="image" src="https://user-images.githubusercontent.com/2896608/87102379-94303100-c206-11ea-94d4-0c701a7a2540.png">

Reply:
That's generally OK to do if you want to it to be blank by default but then required afterwards. I think this was the right move setting the default to an empty string

Reply:
Thanks for chiming in @KalobTaulien !